tests/__init__.py

```diff
@@ -15,7 +16,9 @@
 
 from twisted.trial import util
 
-from tests import utils
+import tests.patch_inline_callbacks
+
+# attempt to do the patch before we load any synapse code
+tests.patch_inline_callbacks.do_patch()
```

Comment:
I wonder if we should print/log that we're replacing inlineCallbacks? Just so that if we accidentally end up importing the tests package there will be some indication that this has happened

Reply:
could do. I'm unconvinced such a warning would be noticed.